zio-schema-json/shared/src/test/scala/zio/schema/codec/JsonCodecSpec.scala

```diff
@@ -436,6 +437,71 @@ object JsonCodecSpec extends ZIOSpecDefault {
           """{"NumberOne":{"value":"foo"}}"""
         )
       }
+    ),
+    suite("Streams")(
+      test("Encodes a stream with multiple integers") {
+        assertEncodesMore(Schema[Int], 1 to 5, charSequenceToByteChunk("1\n2\n3\n4\n5"))
+      },
+      test("Decodes a stream with multiple integers") {
```

Comment:
I would make this a suite and the different checks tests. Easier to read and tests names can give intention

Reply:
Good point, I'll refactor

Reply:
Refactored to suites